### ISSUE TITLE
Input type=file element and SC 1.4.10

### ISSUE THREAD

**Issue by sarasuri** [w3c/wcag#3384]
Hi 

If part of the text associated with the File upload control `<input type="file">` is clipped in reflow, would it fail the SC 1.4.10 Reflow.
![image](https://github.com/w3c/wcag/assets/12256067/a576cd7b-b513-4563-ab26-1bf0b42574e4)

I understand this user interface component is provided by browsers not by authors. Our team is not sure if it is a failure for reflow or not.

Thanks

**Comment by JAWS-test** [comment]
Related: https://github.com/w3c/wcag/issues/1233

1.4.10 unfortunately has no exception for content coming from the browser and not created by the author. This seems to be a defect of the SC.

**Comment by fstrr** [comment]
This looks like the issue can be closed. If it needs to be re-opened, please do so.
